shared/common-adapters/markdown.shared.js

```diff
@@ -30,9 +30,17 @@ function processAST(ast, createComponent) {
   return ast.component
 }
 
+// Quick check to avoid markdown parsing overhead
+const maybeMarkdown = markdown => {
+  // only chars, numbers, whitespace, some common punctuation and periods that end sentences (not domains)
+  return markdown && !markdown.match(/^([A-Za-z0-9!?=+@#$%^&()[\]]|\s|\.\B)*$/)
```

Comment:
@chromakode thanks for catching that. i put in some logging and used the dumb sheet to check it out more. instead of a blacklist going with a whitelist. a simple subset of chars and punc. PTAL